avalanche/training/supervised/strategy_wrappers.py

```diff
@@ -693,6 +694,79 @@ def __init__(
         )
 
 
+class RWalk(SupervisedTemplate):
```

Comment:
to avoid confusion, since RWalk uses also replay, I would either 1) not create this class or 2) introduce also replay in it (it can be provided by the user as input plugin or defined by the strategy if the original paper uses a specific one already in avalanche).

Reply:
Ok, I think I'll remove the class. There's no specified default sampling strategy, although I believe MoF (from ICARL) and simple replay are the most effective in the ablation study. I don't know actually if it suffice to use ICARL+RWalk to have the results in the paper, or if we need to separate MoF from ICARL first (I don't know how the latter works).

Reply:
That would require further examination to understand which kind of replay is best suited to be included as default in the RWalk strategy. You can remove the class for now.